src/templates/layout/base.twig

```diff
@@ -27,6 +27,20 @@
                         <li><a href="{{ url('watch.list') }}">Watch Functions</a></li>
                         <li><a href="{{ url('waterfall.list') }}">Waterfall</a></li>
                     </ul>
+                    {% set sites = sites() %}
```

Comment:
After removing globals usage, I feel like it's better to do it via Twig functions than via Twig globals or injecting in every template.